docs/user-guide/src/developer_environment/tryit.md

```diff
@@ -103,3 +103,501 @@ To set environment variables persistently, export them from the configuration fi
  cp config_example.sh config_$(whoami).sh
  vim config_$(whoami).sh
 ```
+
+## 2. Working with the Development Environment
+
+### 2.1. BareMetalHosts
+
+This environment creates a set of VMs to manage as if they were bare metal
+hosts.
+
+There are two different host OSs that the metal3-dev-env setup process is tested on.
+
+1. Host VM/Server on CentOS, while the target can be Ubuntu or CentOS, Cirros, or FCOS.
+2. Host VM/Server on Ubuntu, while the target can be Ubuntu or CentOS, Cirros, or FCOS.
+
+The way the k8s cluster is running in the above two scenarios is different. For CentOS `minikube` cluster is used as the source cluster, for Ubuntu, a `kind` cluster is being created.
+As such, when the host (where the `make` command was issued) OS is CentOS, there should be three libvirt VMs and one of them should be a `minikube` VM.
+
+In case the host OS is Ubuntu, the k8s source cluster is created by using `kind`, so in this case the `minikube` VM won't be present.
+
+To configure what tool should be used for creating source k8s cluster the `EPHEMERAL_CLUSTER` environment variable is responsible.
+The `EPHEMERAL_CLUSTER` is configured to build `minikube` cluster by default on a CentOS host and `kind` cluster on a Ubuntu host.
+
+VMs can be listed using `virsh` cli tool.
+
+In case the `EPHEMERAL_CLUSTER` environment variable is set to `kind` the list of
+running virtual machines will look like this:
+
+```console
+$ sudo virsh list
+ Id    Name       State
+--------------------------
+ 1     node_0     running
+ 2     node_1     running
+```
+
+In case the `EPHEMERAL_CLUSTER` environment variable is set to `minikube` the list of
+running virtual machines will look like this:
+
+```console
+$ sudo virsh list
+ Id   Name       State
+--------------------------
+ 1    minikube   running
+ 2    node_0     running
+ 3    node_1     running
+```
+
+Each of the VMs (aside from the `minikube` management cluster VM) is
+represented by `BareMetalHost` objects in our management cluster. The yaml
+definition file used to create these host objects is in `bmhosts_crs.yaml`.
```

Comment:
I cannot find the file `bmhosts_crs.yaml`, can you please check if it exists or the name is different ?

Reply:
I can't find it too, i've searched for it on the user-guide but so far the only place it has appeared on is 03_launch_mgmt_cluster.sh script